tests/unittest/_torch/sampler/test_beam_search.py

```
@@ -996,6 +1031,91 @@ def test_smaller_beam_width(
+        llm: LLM,
+        input_prompts: list[str],
+        fixed_params: dict,
+        batch_size: int,
+        sampler_type: str,
+    ):
+        if sampler_type != "TRTLLMSampler":
+            pytest.skip("Test is specific to TRTLLMSampler")
+
+        with pytest.raises(
+                RequestError,
+                match=".*Beam search only supports logprobs when batch size is 1*"
```

Ruff (0.15.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)